src/pages/Home/Home.jsx

```diff
@@ -1,14 +1,27 @@
 import Layout from "../../components/Layout/Layout";
-import { useContext } from "react";
-import UserContext from "../../context/UserContext";
 import Carousel from "../../components/Carousel/Carousel";
 import Card from "../../components/Card/Card";
+import { useEffect, useState } from "react";
 
 const Home = () => {
-  const { user } = useContext(UserContext);
-  if (user === null) {
+  const [businessArr, setBusinessArr] = useState([]);
+
+  const getBusiness = async () => {
+    let response = await fetch(
+      "https://place-api.herokuapp.com/api/v1/places/locationsearch/CH451HE/5"
+    );
+    const data = await response.json();
```

Comment:
you should be able to call the existing `getInRadius` function in placeService which will return you the array of data

Reply:
this way, we can keep all the API requests in the one file